server/statistics/store_collection.go

```diff
@@ -104,6 +104,10 @@ func (s *storeStatistics) Observe(store *core.StoreInfo, stats *StoresStats) {
 
 	// Store flows.
 	storeFlowStats := stats.GetRollingStoreStats(store.GetID())
+	if storeFlowStats == nil {
+		return
```

Comment:
Will skip the CPU statistics？

Reply:
CPU statistics also need rollingStoresStats

Reply:
```go
// GetStoreCPUUsage returns the total cpu usages of threads of the specified store.
func (s *StoresStats) GetStoreCPUUsage(storeID uint64) float64 {
	s.RLock()
	defer s.RUnlock()
	if storeStat, ok := s.rollingStoresStats[storeID]; ok {
		return storeStat.GetCPUUsage()
	}
	return 0
}
```
I think it should not be found in rollingStoresStats.

Reply:
https://github.com/tikv/pd/blob/2c49d5892462928a65206fa363ce8b48aa1e47d2/server/statistics/store.go#L168